src/wallet/rpcwallet.cpp

```diff
@@ -3090,7 +3090,7 @@ UniValue listunspent(const JSONRPCRequest& request)
         if (options.exists("coinType")) {
             int64_t nCoinType = options["coinType"].get_int64();
 
-            if (nCoinType > static_cast<int64_t>(CoinType::MAX_COIN_TYPE)) {
+            if (nCoinType < static_cast<int64_t>(CoinType::ALL_COINS) || nCoinType > static_cast<int64_t>(CoinType::MAX_COIN_TYPE)) {
```

Comment:
Good catch! 👍 But I would rather prefer introducing `MIN_COIN_TYPE = ALL_COINS,` and using it here and in the error message below.

Reply:
Makes sense 👍 changed!